source/docs/advanced/theme/translations.md

```diff
@@ -0,0 +1,93 @@
+---
+id: translations
+title: Translate your application
+---
+
+Big e-commerce websites often need to translate their content in several languages to make sure that they reach a broader range of customers.
+
+In this documentation, we will see how this works for static content in your application.
+
+<blockquote class="note">
+If you want to configure your application to support multiple languages, please refer to [Configure multiple stores](/docs/advanced/production-ready/multistore.html).
+</blockquote>
+
+Front-Commerce manages your translations by using [`react-intl`](https://github.com/yahoo/react-intl), a standard in the React ecosystem. This library works by transforming your static values in React Components. These components will then fetch your translations and display them in your application.
+
+## Declare translations in your application
+
+For instance, let's see how to transform your values in react-intl Components.
+
+* Strings
+```diff
+- Default translation
++ import { FormattedMessage } from "react-intl";
++ <FormattedMessage
++   id="My.Component.Id.messageId"
++   defaultMessage="Default translation"
++ />
+```
+* Dates
+```diff
+- import moment from "moment";
+- moment(date).format("DD/MM/YYYY");
++ <FormattedDate
++   value={props.date}
++   year="numeric"
++   month="long"
++   day="2-digit"
++ />
+```
+* Prices
+```diff
+- <span>{value.amount} €</span>
++ import { FormattedNumber } from "react-intl";
++ <FormattedNumber
++   value={value.amount}
++   style="currency"
++   currency={value.currency}
++ />
+```
+    <blockquote class="note">
+    However in this particular we have abstracted this in Front-Commerce with the [`theme/components/atoms/Price`](https://gitlab.com/front-commerce/front-commerce/tree/develop/src/web/theme/components/atoms/Typography/Price) component.
+    </blockquote>
+* [and many more](https://github.com/yahoo/react-intl/wiki/Components)
+
+### Without React Component
+
+These components will be enough in most cases. However, the issue with these is that it wraps your string within a React Element, which may be troublesome when you actually need to handle the string itself.
+
+For instance, this is the case when you want to add some label attributes to your DOM elements.
+
+```jsx
+<span class="icon" aria-label="Icon title displayed for screen readers" />
+```
+
+Fortunatly, this is correctly handled by react-intl if you use [`defineMessages`](https://github.com/yahoo/react-intl/wiki/API#definemessages) combined with [`injectIntl`](https://github.com/yahoo/react-intl/wiki/API#injection-api).
+
+<blockquote class="wip">
+Please refer to [react-intl documentation](https://github.com/yahoo/react-intl/wiki/API#injection-api) for more information about these two methods. We plan to add an example here in the future but didn't have time to document it yet.
+</blockquote>
+
+## Setup your translations what's in your components
```

Comment:
I really do not see what you wanted to write here… but there definitely seems to be a mistake!